### ISSUE TITLE
Create - Copy activity 

### ISSUE THREAD

**Issue by ChaykeeMor** [ji-devs/ji-cloud#2200]
**Must have a way to paste activity. Currently, in activity kabob menus, advanced options we have 'copy to another JIG'.**

<img width="687" alt="Screen Shot 2022-01-07 at 13 13 05" src="https://user-images.githubusercontent.com/94056434/148535967-abcc0e39-e2b7-4738-b4f1-43b7c9dfcdac.png">

**Then the only way to paste is into a poster activity.  NO WAY FOR USER TO FIGURE THAT OUT and they should be able to paste anywhere..** 

<img width="656" alt="Screen Shot 2022-01-07 at 13 13 17" src="https://user-images.githubusercontent.com/94056434/148536047-8273922b-daae-41c5-a0ac-20e402a05766.png">

What is needed: So in the kabob of any module should take out 'copy to another JIG' and replace with options 'Copy activity' 'Copy design' 'Paste activity' and 'Paste design'. If possible technically, I'd have paste options hidden until you have copied something.

@MendyBerger how doable is this?

**Comment by corinnewo** [comment]
At the very least: 

1. fix up the **paste** so that you can paste into a JIG after any module. Really not intuitive to paste only into cover.
2. hide the paste options until there is a module on the clipboard

Very nice to have, and maybe only after MVP - to copy and paste either design only or whole module (design and activity).

**Comment by MendyBerger** [comment]
@corinnewo I honestly don't think that this should be before the MVP, it's currently not broken, and is clearly an advanced feature. Yes we can make it better, but is it worth doing rather then other tasks?

**Comment by ChaykeeMor** [comment]
@MendyBerger @corinnewo If we are not doing this for the MVP, we should completely hide "advanced options" from the menu until it works.

**Comment by corinnewo** [comment]
At the moment you can ONLY paste to the cover. If you paste from another JIG, you have to go to the cover of the target JIG and paste. But if I want to put the new module in the middle of a jig, cannot. There is only a paste in the cover. I think that there should be a paste in every one of the kebab menus, and the paste will put the copied module below it.

@GaliShapira please confirm before Mendy goes ahead:

1. Add option of "Paste" to every advanced menu in each kebab menu.
2. If there is nothing that has been copied, then the "Paste" button is disabled.
3. If it is on the empty blue module, then it would paste onto the empty module, and a new empty one would appear below.

**Comment by GaliShapira** [comment]
Just to summarize it visualy - these are not yet the final texts, 

**Popup for cover:**
<img width="693" alt="Screen Shot 2022-08-03 at 18 16 41" src="https://user-images.githubusercontent.com/88834002/182645668-f961f4aa-2cc2-4236-9994-6598c9c7a0bc.png">

**Popup for activity:**
<img width="273" alt="Screen Shot 2022-08-03 at 18 16 52" src="https://user-images.githubusercontent.com/88834002/182646257-44a2cb55-0234-42b0-8627-5d10d20513c4.png">

**Popup for the "Drag activity here":**
<img width="705" alt="Screen Shot 2022-08-03 at 18 17 16" src="https://user-images.githubusercontent.com/88834002/182646306-920ececc-eb26-4d55-8503-b45761536dcf.png">

**Popup for publish:**
<img width="518" alt="Screen Shot 2022-08-03 at 18 17 22" src="https://user-images.githubusercontent.com/88834002/182646352-91a73095-9195-405f-aa30-57971b7f5c46.png">

**Alert message when teacher clicked paste when there is already content added in the activity:**
(not designed yet, not final texts)
<img width="389" alt="Screen Shot 2022-08-03 at 18 17 27" src="https://user-images.githubusercontent.com/88834002/182646498-0d966810-f213-4bbf-a6ef-fcb7e62abb6a.png">

**Comment by corinnewo** [comment]
THanks Gali - just a comment for Chaykee - the  **from another JIG**" is not necessary, as you could paste within the same JIG too. It is just what is on the 'clipboard'. Also, Gali, there is currently a DELETE option on the empty module, so I think we need to keep it and add in the extra paste option.

**Comment by GaliShapira** [comment]
I updated my comments, thanks Corinne

**Comment by corinnewo** [comment]
@GaliShapira Chaykee is working on the texts for this. She will be in touch with her corrections.

**Comment by ChaykeeMor** [comment]
Hey @GaliShapira and @corinnewo 

This copy/paste process is a universal function we should not complicate. 'Paste' is never used to mean 'Replace' and I don't think we should make it mean that even if we provide a warning. I understand the need to possibly replace a cover but not an activity so I propose we use the terms 'Copy' 'Paste' [if on empty activity or cover] 'Paste below' [if on designed activity or cover]. This should be a function available at top [not hidden in advanced].

'Advanced' should ONLY appear on activities that are auto-generators [cards] and say 'Use content in:' and drop down with 'flashcards' 'memory game' 'matching' 'multiple choice'

**Comment by corinnewo** [comment]
Thanks @ChaykeeMor. I agree with your comments. It eliminates the need for a warning. @GaliShapira Please update your Zeplin with these texts and instructions, and we can give it to Mendy.

**Comment by GaliShapira** [comment]
So this is what we have. 
It means the copy and paste are not advanced and it makes the menu a bit long... 
Are we ok with that? 

Cover dropdown (with or without content):
<img width="686" alt="Screen Shot 2022-08-04 at 14 45 27" src="https://user-images.githubusercontent.com/88834002/182840299-1820dff9-02d8-4750-a2d3-ac20160a1562.png">

Activity dropdown (with or without content):
<img width="744" alt="Screen Shot 2022-08-04 at 14 54 27" src="https://user-images.githubusercontent.com/88834002/182840820-272c03b9-a434-4a68-b4ef-21f5588eae79.png">

Drag here dropdown:
<img width="695" alt="Screen Shot 2022-08-04 at 14 50 08" src="https://user-images.githubusercontent.com/88834002/182840573-4e72463c-7169-49b4-9d49-f9738850484d.png">

Publish dropdown:
<img width="497" alt="Screen Shot 2022-08-04 at 14 50 26" src="https://user-images.githubusercontent.com/88834002/182840604-f4b3256b-98f4-4e5d-bdd4-414b1662653f.png">

**Comment by ChaykeeMor** [comment]
I don't mind how long it is. But 'Copy' and 'Paste' should be above 'Duplicate'. 'Hide' should be right before delete. and the Advanced text is still wrong. Should be **'Use content in:'** and drop down with 'flashcards' 'memory game' 'matching' **'multiple choice'** [currently still called card quiz]. and Publish should say 'Edit' not 'Edit setting'

**Comment by corinnewo** [comment]
I agree with Chaykee regarding order of items. 
Please also add that if there is nothing copied yet (clipboard empty) then the **Paste** is disabled.
Also, please make sure that it is clear that the Advanced is ONLY for card games. At the moment, all the popups have 'Advanced' and if you click it, there is nothing! Rather not have it at all if it is empty!

**Comment by GaliShapira** [comment]
Thanks for the feedback Chaykee, I updated my comment. 
Hope it can be doable in one ticket...

**Comment by corinnewo** [comment]
Please update the Zeplin for the Publish menu - **Edit** and not **Edit setting**

**Comment by GaliShapira** [comment]
Its updated. Can you check again?

**Comment by corinnewo** [comment]
OK - Thanks! Ready for @MendyBerger !

**Comment by corinnewo** [comment]
Closed in favor of #3252
